src/components/auth/Connect.svelte

```diff
@@ -1,4 +1,18 @@
+<input type="checkbox" id="my-modal-5" checked class="modal-toggle" />
+<div class="modal">
+  <div class="modal-box w-80 relative text-center">
+    <a href="/" class="btn btn-xs btn-circle absolute right-2 top-2">✕</a>
 
-<div>Connect</div>
-
-<a href="/register">Register</a>
+    <div>
+      <h3 class="mb-7 text-xl font-serif">Connect to AppName</h3>
+      <div>
+        <a class="btn btn-primary mb-5 w-11/12" href="/register"
+          >Create a new account</a
+        >
+        <a class="btn btn-primary btn-outline w-11/12" href="/"
+          >I have an existing account</a
+        >
+      </div>
```

Comment:
Ended up just using `margin-bottom` on these components to create space. If you'd like to re-think with `display: grid`, I'm all for it!